cli/bpmetadata/proto/bpmetadata_ui.proto

```diff
@@ -161,11 +163,14 @@ message DisplayVariableToggle {
   string variable_name = 1; // @gotags: json:"variableName,omitempty" yaml:"variableName,omitempty"
 
   // The value of the variable used to toggle the display of another variable.
-  repeated string variable_values = 2; // @gotags: json:"variableValues,omitempty" yaml:"variableValue,omitempty"
+  repeated string variable_value = 2; // @gotags: json:"variableValue,omitempty" yaml:"variableValue,omitempty"
 
   // The type of the variable used to toggle the display of another variable.
   ToggleType type = 3; // @gotags: json:"type,omitempty" yaml:"type,omitempty"
 
+  // The value of the variable used to toggle the display of another variable.
+  repeated string variable_values = 4;  // @gotags: json:"variableValues,omitempty" yaml:"variableValues,omitempty"
+
```

Comment:
@apeabody @njlisi @g-awmalik Changing the field name/number should be fine as this is the exact field names/numbers used within g3.
So syncing this would not make change to g3. PTAL!